apis/logging/v1/input_receiver_types.go

```diff
@@ -6,7 +6,17 @@ package v1
 //
 // The fields of this struct define the set of known Receiver types.
 type ReceiverSpec struct {
```

Comment:
This should have a "type" enum to match the pattern elsewhere to identify which type of input it is

Reply:
Working on it